SwiftLeeds/Views/My Conference/TalkCell.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  TalkCell.swift
+//  SwiftLeeds
+//
+//  Created by Matthew Gallagher on 25/06/2022.
+//
+
+import SwiftUI
+
+struct TalkCell: View {
+    private let time: String
+    private let details: String
+    private let isNext: Bool
+    private let speaker: String?
+    private let company: String?
+    private let gradientColors: [Color]?
+
+    @Environment(\.colorScheme) var colorScheme
+
+    init(time: String, details: String, isNext: Bool = false, speaker: String? = nil, company: String? = nil, gradientColors: [Color]? = nil) {
+        self.time = time
+        self.details = details
+        self.isNext = isNext
+        self.speaker = speaker
+        self.company = company
+        self.gradientColors = gradientColors
+    }
+
+    var body: some View {
+        VStack(spacing: 6) {
+            timeLabel(time)
+
+            VStack(alignment: .leading, spacing: 8) {
+                if let speaker {
+                    VStack(alignment: .leading, spacing: 4) {
+                        Text(speaker)
```

Comment:
We should lean on typography more here, so we can use existing styles to help break up the design and create a visual hierarchy. 

So for instance, we can use .headline, then .subheadline, then .body, with relevant weights. 

It'll look great!

Reply:
I was struggling to get the intent from Figma, so guessed at some of the intent.  As this is just from the current mockup, I would expect the design to change at this stage, so any suggestions are good.  Will be easier to see the full impact once we have data coming into the App so more content to judge the overall appearance on.

Reply:
Yeah, we did say designs aren't pixel perfect they're just descriptive, so its good to experiment a little with whats best. I won't be going in and changing designs at this point unless we have a new feature that we need to consider as we can make any tweaks as needed within our app-work.

Everything is very loosely based on system styles, so the safest best is to use those where logical, as they generally explain themselves in terms of the type hierarchy.

If you'd like, I can run a "design pass" on these after in a separate PR to address things like this